src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -128,9 +130,27 @@ const iconStyles = {
     }
   },
 
-  svg: ({ variables: v }): ICSSInJSStyle => ({
-    fill: v.color,
-  }),
+  svg: getSvgStyle('svg'),
```

Comment:
yes-yes, let me guess what you might think about :) Actually, this is a 'have to' thing, because currently we don't have any possibility to introduce variable set of styles parts, such that, for example, if `props.name` is 'endCall', there will be only parts in styles that are relevant to that icon (`root`, `phone`, 'cross-line', etc). However, there are few things that we should consider before, potentially, making this move:
- if set of styling parts will be decided by values from `props`, it will drastically complicate overall picture - one thing that immediately comes to my mind is increased debugging complexity
- problem of hardcoded style parts - the ones that should fit **_all component from provided set_**, actually, is not that serious: generally there are just handful different parts of the icon that should be styled differently at any given moment, and these parts are entirely theme's aspects: there are no restrictions introduced from core components' functionality on how these style parts should be named, organised or consumed

Reply:
Should we add at least first as many paths properties here as there is in our svg icons so far, and add increase the number of them accordingly if we are adding icons with multiple path parts? Maybe we can generate the paths properties in the begging of the file in order to avoid them being added like this?

Reply:
> Maybe we can generate the paths properties in the begging of the file in order to avoid them being added like this?

yes, but this leads to exact problem I've described above - where the solution is to somehow allow variable amount of component parts being declared for styles (ideally, this number should be dependent on props). While it is an option, there are some drawbacks associated with it - have mentioned them above.

> Should we add at least first as many paths properties here as there is in our svg icons so far, and add increase the number of them accordingly if we are adding icons with multiple path parts

we may consider to do this, yes

Reply:
If we have just as many paths properties as our svg icons actually have, I am okay with them being hardcoded, I see your point. With this, we will be aware about what each of those represent.